homeassistant/components/command_line/switch.py

```diff
@@ -85,10 +89,14 @@ def __init__(
         command_on,
         command_off,
         command_state,
+        icon_template,
         value_template,
         timeout,
     ):
         """Initialize the switch."""
+        super().__init__(
```

Comment:
You cannot use `super()` inheritance here.
You need to implement something like `value_template`